### DISCUSSION TITLE
ImageMagick.MagickCoderErrorException: WriteBlob Failed

### DISCUSSION THREAD

**Discussion by LANSELOT** [dlemstra/Magick.NET#622] [General]
I can't save .png image in Ubuntu

```
...
try
            {
                using (var image = MagickImage(base64Bytes))
                {
                    image.Write(res); // exception here
                }
            }
...
```

**ImageMagick.MagickCoderErrorException: WriteBlob Failed `/storage/watermark/35346472-99c6-4863-98b0-9f9409329296.png' @ error/png.c/MagickPNGErrorHandler/1715**

I used library Magick.NET-Q8-x64.dll

P.S. Write permissions were previously set

**Comment by dlemstra** [comment]
Is it possible to create a small program that demonstrates the issue?
